packages/manager/modules/pci/src/projects/new/config/controller.js

```diff
@@ -18,26 +20,64 @@ export default class PciProjectNewConfigCtrl {
     }).join(', ');
   }
 
-  /* -----  End of Helpers  ------ */
-
-  /* =============================
-  =            Events            =
-  ============================== */
-
-  onProjectConfigFormSubmit() {
-    let setDescrPromise = Promise.resolve(true);
-
+  setCartProjectItem() {
     if (
       this.model.description &&
       !this.cart.projectItem.descriptionConfiguration
     ) {
-      setDescrPromise = this.pciProjectNew.setCartProjectItemDescription(
+      return this.pciProjectNew.setCartProjectItemDescription(
         this.cart,
         this.model.description,
       );
     }
 
-    return setDescrPromise.then(() => this.goToPayment());
+    return this.$q.when();
+  }
+
+  setHdsOptionItem() {
+    const { cartId, projectItem, hdsItem } = this.cart;
+    if (this.hds.isValidSupportLevel) {
+      if (this.model.hds && !hdsItem) {
+        const priceMode = this.hds.option.prices.find(({ capacities }) =>
+          capacities.includes('renew'),
+        );
+        return this.orderCart
+          .addNewOptionToProject(cartId, {
+            duration: priceMode.duration,
+            itemId: projectItem.itemId, // Add HDS option to project
+            planCode: this.hds.option.planCode,
+            pricingMode: priceMode.pricingMode,
+            quantity: 1,
+          })
+          .then((option) => this.cart.addItem(option));
+      }
+      if (!this.model.hds && hdsItem) {
+        return this.orderCart
+          .deleteItem(cartId, hdsItem.itemId)
+          .then(() => this.cart.deleteItem(hdsItem));
+      }
+    }
+
+    return this.$q.when();
+  }
+
+  sendTracking() {
+    this.trackClick('PublicCloud::pci::projects::new_project_continue');
+    if (this.model.hds) {
+      this.trackClick('PublicCloud::pci::projects::new_project_with_HDS');
+    }
+  }
+
+  /* -----  End of Helpers  ------ */
+
+  /* =============================
+  =            Events            =
+  ============================== */
+  onProjectConfigFormSubmit() {
+    this.sendTracking();
+    return this.setCartProjectItem()
+      .then(() => this.setHdsOptionItem())
```

Comment:
What happens if hds option is not supported ? (US for instance)

Reply:
It will be ignored, because hds component is displayed according feature flipping rule.
So, customer can't activate it.
For others customers [EU,CA] they should have the good support level to be able to subscribe to it.